packages/pluggableWidgets/rich-text-web/src/RichText.xml

```diff
@@ -0,0 +1,303 @@
+<?xml version="1.0" encoding="utf-8"?>
+<widget id="com.mendix.widget.custom.richtext.RichText" needsEntityContext="true" pluginWidget="true" offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../../../../node_modules/mendix/custom_widget.xsd">
+    <name>Rich Text</name>
+    <description>Rich inline or toolbar text editing</description>
+    <icon>
```

Comment:
We dont use <Icon> anymore.

Reply:
Missing strudioProCategory and studioCategory

Reply:
Should I remove icon?

Reply:
Yes, it will be a file produced by Ana in the src folder.